test/uploader_spec.js

```diff
@@ -560,7 +560,7 @@ describe("uploader", function () {
       expect(result.moderation[0].kind).to.eql("manual");
     });
   });
-  it.skip("should support requesting ocr analysis", function () {
+  it("should support requesting ocr analysis", function () {
```

Comment:
Don't see any change here, how come it suddenly working?

Reply:
This is due to an account configuration, the account needed an OCR package.

Reply:
Got it